portal/src/locale-data/en.json

```diff
@@ -518,10 +518,11 @@
 
   "PasswordPolicyConfigurationScreen.force-change.label": "Force password change on next login if it does not meet the policy requirements",
   "PasswordPolicyConfigurationScreen.min-length.label": "Minimum password length",
-  "PasswordPolicyConfigurationScreen.require-digit.label": "Requires at least 1 digit",
-  "PasswordPolicyConfigurationScreen.require-lowercase.label": "Requires at least 1 lowercase character",
-  "PasswordPolicyConfigurationScreen.require-uppercase.label": "Requires at least 1 uppercase character",
-  "PasswordPolicyConfigurationScreen.require-symbol.label": "Requires at least 1 symbol",
+  "PasswordPolicyConfigurationScreen.require-digit.label": "Requires at least 1 digit (0-9)",
+  "PasswordPolicyConfigurationScreen.require-lowercase.label": "Requires at least 1 lowercase character (a-z)",
+  "PasswordPolicyConfigurationScreen.require-uppercase.label": "Requires at least 1 uppercase character (A-Z)",
+  "PasswordPolicyConfigurationScreen.require-alphabet.label": "Requires at least 1 alphabetic character (a-z,A-Z)",
```

Comment:
s/alphabetic/alphabet

Reply:
@louischan-oursky It seems it should be "alphabetic character" instead of "alphabet character"